### DISCUSSION TITLE
Implement async select with optional creatable

### DISCUSSION THREAD

**Discussion by cubecleveland** [JedWatson/react-select#4937] [Q&A]
Hello,

Is there a way to call the creatable option when passing a prop to the compoenent that will either enable or disable it?

**Comment by Rall3n** [comment]
Hello @cubecleveland,

there is no prop to enable or disable the `Creatable` functionality directly. But you can hide the new option by using the `isValidNewOption` prop.

```jsx
<AsyncCreatable
  isValidNewOption={(inputValue, value, options, accessors) => {
    if (disableCreatable) return false;

    /* ... */
  }}
/>
```

But be aware that you would have to implement the rest of the `isValidNewOption` logic by yourself if you use this prop.
